solr/modules/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureLogging.java

```diff
@@ -156,20 +150,39 @@ public void testDefaultStoreFeatureExtraction() throws Exception {
         "/response/docs/[0]/=={'fv':'"
             + FeatureLoggerTestUtils.toFeatureVector("store9f1", "3.0")
             + "'}");
+  }
 
-    // Wrong store specified + model specified, should throw exception
-    query.remove("fl");
-    query.add("fl", "fv:[fv store=storeWrong]");
-    assertJQ("/query" + query.toQueryString(), "/error/msg=='Missing feature store: storeWrong'");
+  @Test
+  public void testNonExistentStoreFeatureExtraction() throws Exception {
+    loadFeature("store9f1", ValueFeature.class.getName(), "store9", "{\"value\":3.0}");
+    loadModel(
+        "store9m1",
+        LinearModel.class.getName(),
+        new String[] {"store9f1"},
+        "store9",
+        "{\"weights\":{\"store9f1\":1.0}}");
+
+    final SolrQuery query = new SolrQuery();
+    query.setQuery("id:7");
+    query.add("rows", "1");
+
+    // Non-existent store specified, should throw exception
+    query.add("fl", "fv:[fv store=nonExistentStore]");
+    assertJQ(
+        "/query" + query.toQueryString(), "/error/msg=='Missing feature store: nonExistentStore'");
+
+    // Non-existent store specified + model specified, should throw exception
```

Comment:
split also this, ideally we want just one test to fail if a problem occurs just in one scenario